### PR TITLE
Fixed some French translations

### LINKED ISSUE #1077: Missing some french translation
A lot of translations are missing in French.

### PR DESCRIPTION
**Pull request by pascalheraud**
Here a first pass I did on french translations (fixes #1077). I will continue if this is correct.

### CODE REVIEW

File: web/locales/fr.json [hunk 1]
```diff
@@ -20,19 +20,25 @@
     },
     "country-panel": {
         "source": "source",
+        "renewable": "Renouvelable",
         "carbonintensity": "Intensité carbone",
+        "lowcarbon": "Peu carbonée",
```
Comment (corradio) [inline web/locales/fr.json]: Bas carbone ?

File: web/locales/fr.json [hunk 2]
```diff
@@ -20,19 +20,25 @@
...
     "rankings-panel": {
         "country-ranking": "Classement",
         "show-ranking": "Montrer classement"
     },
     "country-history": {
-        "carbonintensity24h": "Intensité carbone des 24 dernières heures"
+        "carbonintensity24h": "Intensité carbone des 24 dernières heures",
+        "electricityorgin24h": "Origine de l'électricité des 24 dernières heures",
+        "electricityprices24h": "Prix de l'électricité des 24 dernières heures",
+        "Getdata": "Obtenir l'historique and l'API de prévision"
```
Comment (corradio) [inline web/locales/fr.json]: And - > et

File: web/locales/fr.json (hunk 1 above)
Comment (pascalheraud) [reply web/locales/fr.json]: ok

File: web/locales/fr.json (hunk 2 above)
Comment (pascalheraud) [reply web/locales/fr.json]: oops ok

**Pull request merged**